
From: Christoph Hellwig <hch@lst.de>

There seem to be some crazy mac users that want to use the loop driver
on hfsplus.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/hfs/inode.c     |    1 +
 25-akpm/fs/hfsplus/inode.c |    1 +
 2 files changed, 2 insertions(+)

diff -puN fs/hfs/inode.c~hfs-hfsplus-is-missing-sendfile fs/hfs/inode.c
--- 25/fs/hfs/inode.c~hfs-hfsplus-is-missing-sendfile	2004-08-30 00:04:32.463412992 -0700
+++ 25-akpm/fs/hfs/inode.c	2004-08-30 00:04:32.470411928 -0700
@@ -609,6 +609,7 @@ struct file_operations hfs_file_operatio
 	.read		= generic_file_read,
 	.write		= generic_file_write,
 	.mmap		= generic_file_mmap,
+	.sendfile	= generic_file_sendfile,
 	.fsync		= file_fsync,
 	.open		= hfs_file_open,
 	.release	= hfs_file_release,
diff -puN fs/hfsplus/inode.c~hfs-hfsplus-is-missing-sendfile fs/hfsplus/inode.c
--- 25/fs/hfsplus/inode.c~hfs-hfsplus-is-missing-sendfile	2004-08-30 00:04:32.465412688 -0700
+++ 25-akpm/fs/hfsplus/inode.c	2004-08-30 00:04:32.471411776 -0700
@@ -308,6 +308,7 @@ struct file_operations hfsplus_file_oper
 	.read		= generic_file_read,
 	.write		= generic_file_write,
 	.mmap		= generic_file_mmap,
+	.sendfile	= generic_file_sendfile,
 	.fsync		= file_fsync,
 	.open		= hfsplus_file_open,
 	.release	= hfsplus_file_release,
_
